### PR TITLE
Update script to work for Big Sur

### PR DESCRIPTION
**Pull request by nnikolovGC**
Updates scripts to work with macOS 11.

### CODE REVIEW

File: CIS Scripts/3_Security_Remediation.sh [hunk 1]
```diff
@@ -474,37 +472,42 @@ if [ "$Audit2_7_1" = "1" ]; then
 	echo "$(date -u)" "2.7.1 remediated" | tee -a "$logFile"
 fi
 
-# 2.8 Pair the remote control infrared receiver if enabled
+# 2.8 Disable "Wake for network access"
```
Comment (melmols) [inline CIS Scripts/3_Security_Remediation.sh:475]: Curiosity question. Are these rules being renamed or replaced?

**Review by melmols — APPROVED** [review]
Sounds like version updates and some tidying up. LGTM!

File: CIS Scripts/3_Security_Remediation.sh (hunk 1 above)
Comment (nnikolovGC) [reply CIS Scripts/3_Security_Remediation.sh:475]: There are some renames, but most non-functional changes are just rules being moved around by CIS. You can find the different benchmarks at https://www.cisecurity.org/benchmark/apple_os/

**Comment by ghost** [comment]
* As these changes update the previous Catalina benchmark checking that the references to Catalina are replaced with BigSur will be worthwhile.
* Expand upon (README.md)[https://github.com/gocardless/CIS-for-macOS-BigSur-CP/blob/SEC-4964-bigsur/README.md] to better explain how to run and use these scripts.
* Consider setting the CIS scripts with executable perms to make it easier to run the scripts
* Explain why sudo is required to run the checks.
* When running `2_Security_Audit_Compliance.sh` adding a check to validate the MacOS version is the version expected for this script could provide fast feedback if the script is running checks that are for the expected MacOS version

**Pull request merged**